src/SearchBar.js

```diff
@@ -2,40 +2,107 @@ import React, { Component } from 'react';
 import axios from 'axios';
 
 
+        
+//         // randomly pick 3 words and save
+//         // wrap calls in async fucnction? use await to get response from API 1 before we call API 2
+//         // on successful return of API 2, pass saved keywords to GifDisplay component & trigger view switch
+
+//         // ERROR CATCH: when user types in empty string, don't submit the call & prompt user to write a word
+//         // ERROR CATCH: if user types in a string that is not a direct match, return closest possible match
+//             // STRETCH GOAL: instead of closest match, return a list of possible matches and allow the user to pick the one they want
+//         // ERROT CATCH: NO NUMBERS! people work the regex magic
+
+
+
 
 class SearchBar extends Component {
     constructor() {
         super();
         this.state = {
-            placeholder: []
+            movieSearch: [],
+            movieID: [],
+            keywordSearch: [],
+            moviedbAPI: 'b588f737df1d6878d6133a1a7e0bface',
+            giphyAPI: 'NShPdQTfWnvbvgxLo7Jd7C5qDeFfrsLR',
+            userInput: ""
         }
     }
-
-    componentDidMount() {
-    // API CALL 1: movie search based on user's search 
+    getMovie = (event) => {
+        event.preventDefault();
+        // API CALL 1: movie search based on user's search 
         // return the MovieID (also have access to movie details)
-    // API CALL 2: based on MovieID from call 1 
-        // return array of keywords
-        // randomly pick 3 words and save
-    // wrap calls in async fucnction? use await to get response from API 1 before we call API 2
-    // on successful return of API 2, pass saved keywords to GifDisplay component & trigger view switch
-
-    // ERROR CATCH: when user types in empty string, don't submit the call & prompt user to write a word
-    // ERROR CATCH: if user types in a string that is not a direct match, return closest possible match
-        // STRETCH GOAL: instead of closest match, return a list of possible matches and allow the user to pick the one they want
-    // ERROT CATCH: NO NUMBERS! people work the regex magic
-
-    console.log('did mount.')
+        axios({
+            url: 'https://api.themoviedb.org/3/search/movie?',
+            params: {
+                method: `GET`,
+                responseType: `json`,
+                api_key: this.state.moviedbAPI,
```

Comment:
I think we should ask about saving api keys to state. My first thought says we should avoid it because then anyone with react developer tools could access it and steal it. But the same could be said for putting it on github so I'm not sure.